UnityProject/Assets/Scripts/UI/Core/Net/Elements/Dynamic/NetUIDynamicList.cs

```diff
@@ -192,17 +181,10 @@ protected DynamicEntry[] AddBulk(string[] proposedIndices)
 
 		private DynamicEntry PoolSpawnEntry()
```

Comment:
the new body for this method looks like this isn't pooled anymore. Is that intended?

Reply:
yeah, pooling had weird behaviour

Reply:
in that case, the method has to be renamed